BrainPortal/Gemfile

```diff
@@ -46,6 +46,8 @@ group :development do
   gem "pry-doc"
 end
 
+gem 'aws-sdk'
```

Comment:
You don't need to fix this in a new commit, but why insert that gem requirement in the middle of the :development and :test group instead of among the standard list of gem at the top of the file?